src/screens/ChannelListScreenV2/AnonymousChannelListScreen/index.tsx

```diff
@@ -13,6 +13,7 @@ import {ANONYMOUS, ANON_PM, ANON_POST_NOTIFICATION} from '../../../hooks/core/co
 import useAnonymousChannelListScreenHook from '../../../hooks/screen/useAnonymousChannelListHook';
 import useRootChannelListHook from '../../../hooks/screen/useRootChannelListHook';
 import Search from '../../ChannelListScreen/elements/Search';
+import IncognitoEmptyChat from '../IncognitoEmptyChat';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `IncognitoEmptyChat` component is added. Ensure that this component is properly tested and its props are validated using PropTypes or TypeScript interfaces for type safety.
```diff
+ import IncognitoEmptyChat from '../IncognitoEmptyChat';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->